### PR TITLE
Improved position of advanced delivery note

### PR DESCRIPTION
**Pull request by dhairyya**
Making note about advanced delivery more visible

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda could you please approve this for me?

File: microsoft-365/security/office-365-security/tenant-allow-block-list-urls-configure.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ ms.date: 12/05/2022
 
 [!INCLUDE [MDO Trial banner](../includes/mdo-trial-banner.md)]
 
+> [!NOTE]
+>
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list-urls-configure.md]:
```suggestion
```

File: microsoft-365/security/office-365-security/tenant-allow-block-list-urls-configure.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ ms.date: 12/05/2022
 
 [!INCLUDE [MDO Trial banner](../includes/mdo-trial-banner.md)]
 
+> [!NOTE]
+> 
+> - To allow phishing URLs that are part of third-party attack simulation training, use the [advanced delivery configuration](skip-filtering-phishing-simulations-sec-ops-mailboxes.md) to specify the URLs. Don't use the Tenant Allow/Block List.
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list-urls-configure.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**